appraisal.lic

```diff
@@ -25,6 +26,37 @@ class Appraisal
     @equipment_manager.items.each { |item| break unless appraise_item?(item) }
     true
   end
+  
+  def train_appraisal_with_pouches?
+    settings = get_settings
+    return false unless settings.train_appraisal_with_pouches
+    full_pouch_container = settings.full_pouch_container
+    unless full_pouch_container
+      echo "Must set full_pouch_container in YAML"
+      return false
+    end      
+    if full_pouch_container == settings.spare_gem_pouch_container
+      echo "*** Cannot have full pouches be stored where spare pouches are ***"
+      return false
+    end
+    
+    pouches = rummage('C pouch', full_pouch_container)
+    $ORDINALS.each_with_index do |p, i|
```

Comment:
You won't need to rummage anymore, and you can just switch this to `$ORDINALS.each{ |ordinal|` and change `p` to a more meaningful `ordinal`.